src/client/components/DeliveryInfo/DeliveryInfo.component.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './DeliveryInfo.styles.css';
+
+export default function DeliveryInfo({ editMode, vertDisplay, user }) {
```

Comment:
You can destructure the user. Try either:

```suggestion
export default function DeliveryInfo({ editMode, vertDisplay, user: { address, city, zip, country } }) {
```

Alternatively in two lines:

```suggestion
export default function DeliveryInfo({ editMode, vertDisplay, user }) {
  const { address, city, zip, country } = user;
```

Reply:
Changes made as per review comments